spec/service_client/entity_resolver_spec.rb

```diff
@@ -32,16 +32,14 @@
       end
     end
 
-    # rubocop:disable Metrics/LineLength
     context 'when resolver behaves unexpectedly' do
       before do
         stub_request(:get, 'http://127.0.0.1:3001/person?full_name=Wilson,%20Jennifer%20L.&orcid_id=0000-0002-2328-2018')
           .to_return(status: 500, body: '')
       end
-      it 'prints a warning' do
-        expect { resolve }.to output('Error resolving with path person?orcid_id=0000-0002-2328-2018&full_name=Wilson%2C+Jennifer+L.: Entity resolver returned 500 for person type and {"orcid_id"=>"0000-0002-2328-2018", "full_name"=>"Wilson, Jennifer L."} params.' + "\n").to_stderr
+      it 'raises error' do
+        expect { resolve }.to raise_error(RuntimeError)
```

Comment:
add a test that `logger` receives `warn`?

Reply:
@jcoyne Test added.